mods/cnc/languages/chrome/en.ftl

```diff
@@ -696,5 +696,7 @@ button-settings-panel-reset = Reset
 ## tooltips.yaml
 label-latency-tooltip-prefix = Latency:
 label-anonymous-player-tooltip-name = Anonymous Player
+label-bot-player-tooltip-name = Bot
+label-bot-controlled-by-tooltip = Bot controlled by { $name }
```

Comment:
"Controlled" sounds like the human player can give them orders in game to me, maybe a different word like "Bot managed by { $name }"? Or maybe even just "Bot" or "AI Player".

Or I could be overthinking it.

Reply:
I had similar thoughts, but was uncreative.